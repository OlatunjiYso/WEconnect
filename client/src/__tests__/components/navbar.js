import React from "react";

import Navbar from '../../components/navbar';

describe('All tests for navbar component', () => {
 
  const props = {
    loggedIn: '',
    logout: ()=> {}
  };
  
    it('should be defined', () => {
      expect(Navbar).toBeDefined();
    });
    it('should render the navbar', () => {
      const { wrapper } = shallow(<Navbar { ...props }/>);
      expect(wrapper).toMatchSnapshot();
    });
});
