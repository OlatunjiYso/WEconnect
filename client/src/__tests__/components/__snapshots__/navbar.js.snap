// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for navbar component should render the navbar 1`] = `undefined`;
